packages/ihe-gateway/server/Channels/CareQuality Document Exchange - Outbound/XCPD ITI-55 Processor/destinationConnector-XCPD Endpoint-responseTransformer-step-0-Process response.js

```diff
@@ -3,6 +3,12 @@ var http = $('responseStatusLine');
 http = String(http).replace('HTTP/1.1 ', '').replace(/\D/g, '');
 channelMap.put('HTTP', http.toString());
 
+// https://docs.nextgen.com/bundle/Mirth_User_Guide_4_4_2/page/connect/connect/topics/c_Common_Scenarios_connect_ug.html
+//if (responseStatus == QUEUED && connectorMessage.getSendAttempts() >= 2) {
+//	responseStatus = ERROR;
+//	responseStatusMessage = 'Maximum send attempts exceeded.';
+//}
+
```

Comment:
Leftover?

Reply:
I left it there on purpose b/c I couldn't make it to work but that would be the ideal setup if I understand this correctly; i.e., the dest connector has its own threads and doesn't block the channel.